middleware/local_access.py

```
@@ -0,0 +1,101 @@
+
+logger = logging.getLogger(__name__)
+
+
+class LocalAccessMiddleware(BaseHTTPMiddleware):
+    """
+    로컬 IP 주소에서의 접근만 허용하는 미들웨어
+
+    Starlette의 BaseHTTPMiddleware를 상속받아 구현
+    """
+
+    ALLOWED_LOCAL_IPS = ["127.0.0.1", "0.0.0.0", "localhost"]
```

Ruff (0.14.8)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)